Header header
uint64 Id
geometry_msgs/Transform T_SC
geometry_msgs/Transform T_WS
sensor_msgs/Image descriptors
sensor_msgs/Image keypoints
okvis_ros/keypoint[] keypoints_new
float64 distance